packages/dialog/stories/index.stories.mdx

```diff
@@ -76,6 +77,64 @@ The documentation of the full config object can be found in the `lion/overlay` p
 
 The `config` property uses a setter to merge the passed configuration with the current, so you only **overwrite what you pass** when updating `config`.
 
+### Styling content
+
+Because the content of a dialog is slotted, it can only be styled one level deep. If you need more, you should create a custom component that passes the styles along with the html.
```

Comment:
It's actually impossible to style it from the `dialog` component class, because the content slot is moved to the global root node. This is why you should create a custom component and slot it in as the content, this ensures style encapsulation on your dialog content.